domains/pclzmb.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "ZombieRBX",
+           "email": "",
+           "discord": "1287108529357979709",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.lT0Uhhox-qjcDTBTbJ8oDwE6tPb_yb2SEzSf3q3n7jsP5o5gvJVP1jRaoB7v5hQwFhKe1wOVjLukFqWOF4pq1Pgaj6OkiBtObbp58nlQorsCO2uALEdAmrQjNHkYeEZdAZRftN79bgHTOCpKWLgPXOVYqKQPST_XHmt4Pf1geDi9uJLHYyDJBsSk6f-8r0HVEImMltrBSxq1LzuoiTSNGloQ_fs37t7qRq7VRleJ3D95lFkgtNcQDU_A1zRJAfMZYiA14hYbw6bAnXP2f5LKK15Y02OopKjig-TATxVmfatOdP_vr6dtBzKiGfwpYE81fh5L8EZlfAKkUZtc2eb6Aw.DjbHit8xBRFuhx0r0tBN-A.E1lsqQze1wQVJs-W-XdvERmpJlRX8IY7NAGtSmp6ykm4TdD_N_2E1nZCwARt-1PDWEBsY8lWaYQ8II-Mk17ccImFt7kgQK2TVx0MhBpoz3M.rszpDurL3zG0bx440pYrDQ"
+        },
+    
+        "record": {
+            "CNAME": "https://zombierbx.github.io/peacefullyzombieonyt.net/"
```

Comment:
Invalid CNAME, should be a hostname with no protocols or file paths. See the documentation for information: https://docs.is-a.dev